rs/nervous_system/agent/src/agent_impl.rs

```diff
@@ -32,12 +32,13 @@ impl CallCanisters for Agent {
                 .with_arg(request_bytes)
                 .call()
                 .await?;
-            match request {
+            let (response, _cert) = match request {
                 ic_agent::agent::CallResponse::Response(response) => response,
                 ic_agent::agent::CallResponse::Poll(request_id) => {
                     self.wait(&request_id, canister_id).await?
                 }
-            }
+            };
+            response
```

Comment:
Since it looks like this is only used in pocket_ic, it is OK not to check the cert. However, it would better to note this somehow in the documentation or in the function name (e.g. call_insecure) so it isn't used in the wrong context.

Reply:
UpdateCall.call in fact checks the cert. (This is not mentioned in the doc string.)